app/controllers/placements/support/support_users_controller.rb

```diff
@@ -6,27 +6,16 @@ def index
   end
 
   def new
-    @support_user = if params[:support_user].present?
-                      Placements::SupportUser.new(support_user_params)
-                    else
-                      Placements::SupportUser.new
-                    end
+    @support_user_form = params[:support_user].present? ? support_user_form : SupportUserInviteForm.new
   end
 
   def check
-    @support_user = Placements::SupportUser.new(support_user_params)
-
-    render :new unless @support_user.valid?
+    render :new unless support_user_form.valid?
   end
 
   def create
-    @support_user = Placements::SupportUser.new(support_user_params)
-
-    if SupportUser::Invite.call(support_user: @support_user)
-      redirect_to placements_support_support_users_path, flash: { success: t(".success") }
-    else
-      render :check
-    end
+    support_user_form.invite!
```

Comment:
We can now call `save!` here.

```suggestion
    support_user_form.save!
```

Reply:
resolved in [ac16cba](https://github.com/DFE-Digital/itt-mentor-services/pull/253/commits/ac16cba008ff65bed305d3cff34dce4ba115f505)